src/czmq_selftest.c

```diff
@@ -50,6 +50,7 @@ all_tests [] = {
     { "zdir_patch", zdir_patch_test, true, true, NULL },
     { "zfile", zfile_test, true, true, NULL },
     { "zframe", zframe_test, true, true, NULL },
+    { "zprint", zframe_print_test, true, true, NULL },
```

Comment:
the model is one test function per class, please fold this into the existing zframe_test function or it will be overwritten when regenerating the code from zproject

Reply:
Thanks; done.